pages/02.Documentation/02.compute/docs.de.md

```diff
@@ -16,11 +16,11 @@ Sowohl via unserer öffentlichen [OpenStack API](../../03.Tutorials/04.api-acces
 
 ## Flavors
 
-### Standard network storage instance types (M1)
+### Standard Instanz-Typen (M1)
 
-Standard instances generally offer you good performance, availability and storage durability. Disk data will be distributed across multiple servers (SysEleven Distributed Storage).
-
-We recommend these instance types for most workloads and applications.
+Standard Instanz-Typen sind mit einem verteiltem Root-Speichermedium einer festen Größe ausgestattet.
```

Comment:
den satz würde ich bei `Sie können den Root-Speicher durch unseren ebenfalls verteilten Block-Speicher ergänzen.`

Reply:
ich finde es besser mit "ausgewogene performance" etc. + wir empfehlen das für die meisten Anwendungen und Nutzungsfälle.

Reply:
What you are missing seems to be in the next line. Please compare stance by stance instead of line by line.

Reply:
Let's try a different order of the sentences :) I liked the general overview the old first sentence gave you.